README.md

```diff
@@ -15,10 +15,13 @@
 
 <!--ABOUT ME CODE-->
 <h3 align="left">👨🏻‍💻  About Me</h3>
-🎓  I am currently pursuing a B.Tech in Information Technology at Shri Ram Institute of Technology.<br>
+👋  Hi, I'm Ajay Patel, working at Hexaview Technology with 2️⃣ years of experience as a software developer.<br> 
```

Comment:
Hi, I'm Ajay Patel, a software developer at Hexaview Technologies with 2️⃣ years of professional experience.